bundle-workflow/python/git/git_repository.py

```diff
@@ -25,6 +25,6 @@ def __post_init__(self):
         self.sha = subprocess.check_output(['git', 'rev-parse', 'HEAD'], cwd = self.dir.name).decode().strip()
         print(f'Checked out {self.url}@{self.ref} into {self.dir.name} at {self.sha}')
 
-    def execute(self, command):
+    def execute(self, command, dirname=""):
```

Comment:
This looks like a subdirectory. Let's call it `subdirname` or something like that? 

Would it work with `None` instead of `""`? From the interface POV it's strange to pass an empty string when otherwise we mean `None`. Maybe just `if dirname: ...join` to make it explicit.

Nit: most code is styling with spaces around `=`, I know we want some kind of automatic linting, but in the meantime let's try to make things at least look a little consistent?